.github/workflows/test-sdks-remote.yml

```diff
@@ -4,9 +4,30 @@ on:
   push:
     branches:
       - main
+      - tp/workflows/round2
```

Comment:
temporary I'm guessing?

What about running these shared tests on all pull requests, like we do for the individual SDKs?

Reply:
`test-sdks.yml` is for in-flight changes; we don't want to make the SDKs _in their respective repos_ until the test cases are checked in.

Temporary indeed; removed.